docs/package_firewall/pypi.md

```diff
@@ -51,6 +51,9 @@ ERROR: Could not find a version that satisfies the requirement <malicious> (from
 ERROR: No matching distribution found for <malicious>
 ```
 
+If a verison range is accepted by the manifest, the package manager will
+automatically attempt using a version that passes Phylum's policy.
```

Comment:
I've added this to all ecosystems because I thought it's worth pointing out that under **normal** operation, the user shouldn't really see much at all. I think that's a bit nicer than telling the user to just expect errors whenever trying to do anything.

For PyPI it's a bit unfortunate because there's two documented package managers, but with the current structure I think duplicating it is either than trying to restructure things.

Reply:
`verison` should be `version` here and in the other places this text was added.

Reply:
You sure? I'm pretty sure it's verison. :)